utils/diags.sh

```diff
@@ -7,15 +7,17 @@ then
 fi
 
 echo "Collecting diags"
+echo "This script attempts to collect a wide range of diagnostics, some of which may not be available on your system - don't worry if this script produces warning messages for these."
 
 ROUTE_FILE=route
 CALICO_CFG=/etc/calico
 CALICO_DIR=/var/log/calico
 NEUTRON_DIR=/var/log/neutron
+NOVA_DIR=/var/log/nova
 date=`date +"%F_%H-%M-%S"`
-diags_dir=/tmp/"$date"
+diags_dir=./"$date"
 system=`hostname`
-echo "  created dir $diags_dir"
+echo "  creating dir $diags_dir"
 mkdir "$diags_dir"
 pushd "$diags_dir" > /dev/null
```

Comment:
I think we care greatly if the mkdir or pushd commands fail: recommend bailing out in those cases.